docs/docs/chitchat-faqs.mdx

```diff
@@ -0,0 +1,338 @@
+
+We’re going to use content from [Sara](https://github.com/RasaHQ/rasa-demo), the Rasa
+assistant that, amongst other things, helps the user get started with the Rasa products.
+You should first install Rasa using the [Step-by-step Installation Guide](./installation.mdx#step-by-step-installation-guide)
+and then [prototype an assistant](./prototype-an-assistant.mdx)
+to make sure you know the basics.
+
+To prepare for this tutorial, we’re going to create a new directory and start a
+new Rasa project.
+
+```bash
+mkdir rasa-assistant
+rasa init
+```
+
+Let’s remove the default content from this bot, so that the `data/nlu.md`, `data/stories.md`
+and `domain.yml` files are empty.
+
+<a aria-hidden="true" tabIndex="-1" className="anchor enhancedAnchor" id="respond-with-memoization-policy"></a>
+
+## Memoization Policy
+
+The MemoizationPolicy remembers examples from training stories for up to a `max_history`
+of turns. The number of “turns” includes messages the user sent, and actions the
+assistant performed. For the purpose of a simple, context-less FAQ bot, we only need
+to pay attention to the last message the user sent, and therefore we’ll set that to `1`.
+
+You can do this by editing your `config.yml` file as follows (you can remove `TEDPolicy` for now):
+
+```yaml
+policies:
+- name: MemoizationPolicy
+  max_history: 1
+- name: MappingPolicy
+```
+
+:::note
+The MappingPolicy is there because it handles the logic of the `/restart` intent,
+which allows you to clear the conversation history and start fresh.
+
+:::
+
+Now that we’ve defined our policies, we can add some stories for the `goodbye`, `thank` and `greet`
+intents to the `data/stories.md` file:
+
+```md
+## greet
+* greet
+  - utter_greet
+
+## thank
+* thank
+  - utter_noworries
+
+## goodbye
+* bye
+  - utter_bye
+```
+
+We’ll also need to add the intents, actions and responses to our `domain.yml` file in the following sections:
+
+```md
+intents:
+  - greet
+  - bye
+  - thank
+
+responses:
+  utter_noworries:
+    - text: No worries!
+  utter_greet:
+    - text: Hi
+  utter_bye:
+    - text: Bye!
+```
+
+Finally, we’ll copy over some NLU data from Sara into our `data/nlu.md` file
+(more can be found [here](https://github.com/RasaHQ/rasa-demo/blob/master/data/nlu/nlu.md)):
+
+```md
+## intent:greet
+- Hi
+- Hey
+- Hi bot
+- Hey bot
+- Hello
+- Good morning
+- hi again
+- hi folks
+
+## intent:bye
+- goodbye
+- goodnight
+- good bye
+- good night
+- see ya
+- toodle-oo
+- bye bye
+- gotta go
+- farewell
+
+## intent:thank
+- Thanks
+- Thank you
+- Thank you so much
+- Thanks bot
+- Thanks for that
+- cheers
+```
+
+You can now train a first model and test the bot, by running the following commands:
+
+```bash
+rasa train
+rasa shell
+```
+
+This bot should now be able to reply to the intents we defined consistently, and in any order.
+
+For example:
+
+
+
+<img alt="Memoization Policy Conversation" src="../img/memoization_policy_convo.png" />
```

Comment:
we need to use `useBaseUrl()` for this to work in all the situations (dev build, production), same for all content that is under the `static/` folder. You can find examples of this in other documents :)